apps/test/unit/code-studio/components/WorkspaceAlertTest.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import {expect} from '../../../util/deprecatedChai';
+import {mount} from 'enzyme';
+import WorkspaceAlert from '@cdo/apps/code-studio/components/WorkspaceAlert';
+import $ from 'jquery';
+import sinon from 'sinon';
+
+describe('WorkspaceAlert', () => {
+  it('can be at the top or bottom', () => {
+    var jQueryHeight = sinon.stub($.fn, 'height').returns(4);
```

Comment:
Sometimes a stub can cause issues when run in parallel with other unit tests in our test engine. You can prevent this by calling `$.fn.height.restore()` at the end of each test where you use this stub.